Schema/generateSchema.ps1

```diff
@@ -0,0 +1,60 @@
+$generatorPath = "C:\src\mseng\AppInsights-Common"
+$publicSchemaLocation = "https://raw.githubusercontent.com/Microsoft/ApplicationInsights-Home/master/EndpointSpecs/Schemas/Bond"
+
+
+$currentDir = $scriptPath = split-path -parent $MyInvocation.MyCommand.Definition
+#fix path
+$generatorPath = "$generatorPath\..\bin\Debug\BondSchemaGenerator\BondSchemaGenerator"
+
+
+#####################################################################
+## PUBLIC SCHEMA
+#####################################################################
+
+mkdir -Force $currentDir\PublicSchema
+
+del "$currentDir\PublicSchema\*.bond"
+
+$argumentList = ""
+
+# Download public schema from the github
+@(
+"AvailabilityData.bond",
```

Comment:
Would it be possible to automatically get all *.bond files from a Schema directory?